src/components/AppBar/AppBar.tsx

```diff
@@ -84,6 +34,8 @@ const _AppBar = () => {
   const locationPath = useLocation();
   const { isEmbed } = useEmbed();
 
+  const isLocationPage = location.pathname.includes('us');
```

Comment:
If you want to detect the homepage, you can probably use `location.pathname.length === 1`

Reply:
re: our convo on slack, this wouldnt work :(